packages/kit/src/vite/preview/index.js

```diff
@@ -42,6 +42,12 @@ export async function preview(vite, config, protocol) {
 	});
 
 	const server = new Server(manifest);
+	server.init({
+		// This just makes sure no values are undefined
+		env: Object.entries(process.env).reduce((prev, [k, v]) => {
+			return { ...prev, [k]: v ?? '' };
+		}, {})
```

Comment:
same question as above

```suggestion
		env: process.env
```